website/content/docs/ecosystem.mdx

```diff
@@ -6,194 +6,86 @@ description: Comparison between Nomad and Kubernetes
 
 # Nomad Ecosystem
 
-### Continuous Integration & Delivery
+## Continuous Integration & Delivery
 
-#### GitLab
+- [GitLab](https://www.hashicorp.com/resources/nomad-ci-cd-developer-workflows-and-integrations)
+- [Codefresh](https://codefresh.io/docs/docs/yaml-examples/examples/nomad/)
+- [CircleCI](https://circleci.com/docs/2.0/nomad/)
```

Comment:
This link is showing an old version of Nomad that they're saying is deprecated. Let's update this link:

```suggestion
- [CircleCI](https://circleci.com/docs/server/operator/introduction-to-nomad-cluster-operation)
```